src/resources.js

```diff
@@ -2,14 +2,14 @@ import Vue from 'vue';
 
 Vue.use(require('vue-resource'));
 
-/*
+
 let protocol = "http";
 let hostname = "localhost";
-let port = "5600";
-//let port = "5666";
+//let port = "5600";
+let port = "5666";
 let origin = protocol + "://" + hostname + ":" + port;
- */
-let origin = "";
+
+//let origin = "";
```

Comment:
This is exactly what breaks so you cannot use both 127.0.0.1 and localhost. Before we simply used origin="" so it chose 127.0.0.1/localhost depending on what's in the URL so no CORS issues occur, but when hardcoding localhost you cannot use 127.0.0.1